src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 800px;
+  padding: 10px;
+
+  @extend %start-day-sun;
+
+  @for $i from 1 through $month-length {
```

Comment:
The variable `$month-length` is used in the loop, but it's not defined in this file. Make sure that `$month-length` is defined and available in the scope where this SCSS is compiled.